.github/workflows/build-and-test.yaml

```diff
@@ -16,6 +16,31 @@ env:
   main_go_path: .
 
 jobs:
+  test-buildah:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install buildah
+        run: sudo apt update && sudo apt install -y podman buildah qemu-user-static
+
+      - name: Build container amd64
```

Comment:
Rh has already a action for this: https://github.com/redhat-actions/buildah-build#building-using-containerfiles

Reply:
I wanted to try not using actions but commands and see if it's more readable or not :)

Reply:
then you should have also the action as comparison in the code 😄 ... think this should replace both run actions.

```yaml
    - name: Buildah Action
      uses: redhat-actions/buildah-build@v2
      with:
        image: db-operator
        archs: amd64, arm64
        tags: "latest ${{ github.sha }}"
```

Reply:
I meant actions in general. I was thinking to check more scripts-like approach instead of referencing to some actions, because I didn't like figuring out what actions should've been updated to which versions when github did some deprecations. But maybe we should stick to actions, because running scripts is also not that great :)